pkg/network/ebpf/c/protocols/postgres/decoding.h

```diff
@@ -260,7 +305,14 @@ int uprobe__postgres_tls_process(struct pt_regs *ctx) {
         bpf_tail_call_compat(ctx, &tls_process_progs, PROG_POSTGRES_PROCESS_PARSE_MESSAGE);
         return 0;
     }
-    postgres_entrypoint(pkt, &tup, &header, (__u8)args->tags);
+    __s8 result = postgres_entrypoint(pkt, &tup, &header, (__u8)args->tags);
+
+    postgres_kernel_msg_count_t* pg_msg_counts = get_pg_msg_counts_map(pkt);
+    if (pg_msg_counts != NULL) {
```

Comment:
redundant check

Reply:
I worried that verifier will fail without this condition, I've changed this code as well.